wiki/Ranking_Criteria/osu!mania/de.md

```diff
@@ -1,91 +1,206 @@
----
-outdated_translation: true
----
-
 # osu!mania Ranking Kriterien
 
-Merke: Die Osu!Mania Ranking Kriterien beinhalten die kürzlichen Änderungen [durch das Diskussions-Forum](https://osu.ppy.sh/community/forums/87). Jede Regel, die dort diskutiert wird, ist hier so aufgelistet, wie sie momentan gilt und wird erst aktualisiert, wenn die Diskussion eine Einigung erreichen konnte.
+Die **osu!mania Ranking Kriterien** sind Regeln und Richtlinien, welche für die Gestaltung von osu!mania-spezifischen Schwierigkeitsstufen verwendet werden. Um eine osu!mania-spezifische Schwierigkeitsstufe zu ranken, ist es verpflichtend, dass die Gestaltung sich an die aufgelisteten Kriterien hält. Während **alle Regeln in jedem Fall befolgt werden müssen**, so können Richtlinien unter besonderen Umständen ignoriert werden. Diese besonderen Umstände müssen durch eine ausführliche Erklärung begründet werden, warum diese Richtlinie missachtet wurde und warum die generelle Qualität der Gestaltung gestört wird, wenn diese Richtlinie nicht ignoriert wird.
+
+Regelvorschläge sowie Anregungen werden im [Diskussionsforum für die Ranking Kriterien](/wiki/Ranking_Criteria/Difficulty_naming) besprochen. Jegliche Regel, die eine Diskussion und den Genehmigungsprozess der Community durchlaufen ist, wird hier aufgeführt, da sie in den jeweiligen Diskussionsfäden akzeptiert wurde.
 
-## Übliche Bedingungen
+## Glossar
 
 ### Schwierigkeitsgrade
 
-- **EZ** = Einfach (Beginner/Easy/Basic)
-- **NM** = Normal (Light/Normal/Novice)
-- **HD** = Hard (Hyper/Advanced)
-- **MX** = Insane (Another/Exhaust)
-- **SC/SHD** = Extra (Black Another/Expert/Infinite/Gravity)
+*Hauptartikel: [Difficulty Naming](/wiki/Ranking_Criteria/Difficulty_naming)*
+
+- ![](/wiki/shared/diff/easy-m.png?20211215) Easy
+- ![](/wiki/shared/diff/normal-m.png?20211215) Normal
+- ![](/wiki/shared/diff/hard-m.png?20211215) Hard
+- ![](/wiki/shared/diff/insane-m.png?20211215) Insane
+- ![](/wiki/shared/diff/expert-m.png?20211215) Expert
+
+### Gebräuchliche Begriffe
+
+- **BPM**: Tempo von einem Song, gemessen in Beats pro Minute.
+- **Column**: Vertikale Linien auf dem Spielfeld, auf dem die Noten platziert werden. Der Editor benutzt `0|1|2|3|4|5|6|7|8|9` für Zeitstempel Bezeichnungen.
+- **Key mode**: Die Anzahl der Spalten in einer Schwierigkeitsstufe.
+- **Keysounding**: Hitsounding an jeweiliger Stelle mit Hilfe von Stichproben, welche sehr ähnlich oder direkt vom Song stammen.
```

Comment:
```suggestion
- **Keysounding**: Hitsounding an jeweiliger Stelle mit Hilfe von Samples, welche sehr ähnlich oder direkt vom Song stammen.
```
I think using `Samples` here is fine, as it is defined in the [Duden](https://www.duden.de/rechtschreibung/Sample)